packages/grafana-e2e/src/index.ts

```diff
@@ -13,6 +13,7 @@ import { getScenarioContext, setScenarioContext } from './support/scenarioContex
 import * as typings from './typings';
 
 const e2eObject = {
+  // eslint-ignore-next-line
```

Comment:
what rule are we ignoring here? betterer shouldn't be concerned about these lines, and it doesn't look like you changed anything else related to them?

Reply:
This has been reverted 😊